libpod/container_internal_linux.go

```diff
@@ -482,6 +482,19 @@ func (c *Container) checkpoint(ctx context.Context, options ContainerCheckpointO
 	if c.state.State != ContainerStateRunning {
 		return errors.Wrapf(ErrCtrStateInvalid, "%q is not running, cannot checkpoint", c.state.State)
 	}
+
+	// Create the CRIU log file and label it
+	dumpLog := filepath.Join(c.bundlePath(), "dump.log")
+
+	logFile, err := os.OpenFile(dumpLog, os.O_RDWR|os.O_CREATE, 0600)
+	if err != nil && os.IsNotExist(err) {
```

Comment:
why are other errors ignored?

Reply:
I think I got confused by the flags in OpenFile().

The goal is to have a file independent if it exists or not. There just needs to be a file so that it can be labelled. I changed it to use O_CREATE and now it works if the file already exists or not.